content/blog/2021/04/2021-04-20-tekton-plugin.adoc

```diff
@@ -0,0 +1,182 @@
+---
+layout: post
+title: "Easily reuse Tekton and Jenkins X from Jenkins"
```

Comment:
or "Easily reuse Tekton from Jenkins and Jenkins X"? It does not look like we reuse Jenkins X from Jenkins, but I might be wrong

Reply:
the title is correct, we can reuse Tekton and/or Jenkins X (pipelines) from Jenkins